### DISCUSSION TITLE
Smartwatches that provide access to their barometric sensor?

### DISCUSSION THREAD

**Discussion by dennisguse** [OpenTracksApp/OpenTracks#1068] [General]
@vanous do you know if there are smartwatches (ones that are supported by Gadgetbridge) that provide barometric data via BLE?
I will get a new phone and it doesn't have a barometric sensor, but I need one...

**Comment by vanous** [comment]
Hmm,

even my original Bip has altitude and pressure but these are not exposed to ble... and are only available when the GPS chip is running. So to be honest, I really don't know of any device. I will ask around and will let you know.

**Reply by dennisguse** [reply]
Thanks!

**Reply by vanous** [reply]
Interestingly enough, look at this PR which we are about to merge:

https://codeberg.org/Freeyourgadget/Gadgetbridge/pulls/2564

`Read atmospheric pressure from device`. But i am not sure if this can be observed also on earbuds...

**Reply by dennisguse** [reply]
Thx! Does not really fit to my plan as I need the barometer while cycling (and want to wear a cycling helmet) ;)
